api.rst

```diff
@@ -1148,13 +1148,13 @@ PostgreSQL has four procedural languages that are part of the core distribution:
 Immutable and stable functions
```

Comment:
I tried to rephrase that a bit to make it more clear what is actually happening (the volatility marker has little real effect - except for POST actually), but I'm not sure I'm 100% happy yet. @steve-chavez what do you think?

Reply:
The GET case looks better. However POST doesn't look that clear(gets a bit lost).

I think POST deserves it's own paragraph. Could be like: "For POST pgrst executes the statement in a write(read/write) transaction except for functions marked as immutable/stable...".

Reply:
I think I got it now. What do you think?

Reply:
It's clearer. I like it :+1: